### PR TITLE
Sc 237 setup the project

### PR DESCRIPTION
**Pull request by marialobillo**

### CODE REVIEW

File: server/node_modules/.bin/mime
Comment (AbdelrahmanRadwan) [inline server/node_modules/.bin/mime:1]: you shouldn't push the node modules or any build files.

File: server/.gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+node_modules
```
Comment (AbdelrahmanRadwan) [inline server/.gitignore]: In your MR, they are not ignored.

File: server/.gitignore [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+build
+
```
Comment (AbdelrahmanRadwan) [inline server/.gitignore:9]: Here is an example of the js, node.js gitignore for your reference.
https://github.com/github/gitignore/blob/main/Node.gitignore

File: server/.gitignore (hunk 1 above)
Comment (marialobillo) [reply server/.gitignore]: Good point

File: server/.gitignore (hunk 2 above)
Comment (marialobillo) [reply server/.gitignore:9]: I will added this one you share, thanks :)

**Pull request merged**